workflow/executor/os-specific/umask_darwin.go

```diff
@@ -0,0 +1,7 @@
+package os_specific
+
+import "syscall"
+
+func CallUmask(mask int) (oldmask int) {
```

Comment:
minor - you ignore the return value and you only ever pass in zero - so how about:

```
func UmaskZero() {
// ...
}
```

What might be even better is name it after you intent.

Reply:
```
GrantWriteAccessToNonRootUsers
```

??

Reply:
What do you think about AllowGrantingAccessToEveryone?

Reply:
OK, updated